app/assets/stylesheets/modules/_landing.scss

```diff
@@ -38,4 +38,25 @@
     margin: 20px 0;
     text-transform: uppercase;
   }
+
+  .landing-section {
+    margin: 50px auto;
+  }
+
+  .landing-graphic {
+    width: 100%;
+  }
+
+  .landing-graphic-medium {
+    display: block;
+    margin: 0 auto;
+    width: 70%;
+  }
+
+  .contribute-repo-logo {
```

Comment:
lets wrap this within the namespace since the styles are specific to the landing page. 
`.landing-contribute-repo-logo`

WDYT?